sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/ManagedClustersGetCommandResultHeaders.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.resourcemanager.containerservice.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.http.HttpHeaderName;
+import com.azure.core.http.HttpHeaders;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The ManagedClustersGetCommandResultHeaders model. */
+@Fluent
+public final class ManagedClustersGetCommandResultHeaders {
```

Comment:
Why do we have these? Are it LRO?

Reply:
They have 202 in their 2022-09 version `getCommandResult`, but no header: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2022-09-01/managedClusters.json#L1950-L1952

Reply:
Ok. Maybe just garbage out... So be it. Fine with me.

I would expect a GET not respond a 202.